packages/account/src/Sections/Security/Passkeys/components/passkeys-status.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Icon, Text } from '@deriv/components';
+import FormBody from '../../../../Components/form-body';
+
+type TPasskeysStatus = {
+    title: React.ReactElement;
+    description?: React.ReactNode;
+    icon: string;
+    className?: string;
+};
+
+const PasskeysStatus = ({
+    title,
+    description,
+    icon,
+    children,
+    className,
+}: React.PropsWithChildren<TPasskeysStatus>) => {
+    return (
+        <React.Fragment>
+            <FormBody
+                scroll_offset='22rem'
+                className={classNames('passkeys-status__wrapper', {
+                    [`${className}`]: className,
+                })}
+            >
+                <Icon icon={icon} size={96} />
+                <Text as='div' color='general' weight='bold' size='s' align='center' className='passkeys-status__title'>
```

Comment:
I think by default size is 's'. We can remove the `size='s'`

Reply:
yes, indeed. missed it. Thanks @suisin-deriv